rocAL/rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -112,11 +125,7 @@ def image(*inputs, user_feature_key_map=None, path='', file_root='', annotations
     return (decoded_image)
 
 
-def image_raw(*inputs, user_feature_key_map=None, path='', file_root='', annotations_file='', shard_id=0, num_shards=1, random_shuffle=False,
-              affine=True, bytes_per_sample_hint=0, cache_batch_copy=True, cache_debug=False, cache_size=0, cache_threshold=0, cache_type='',
-              device_memory_padding=16777216, host_memory_padding=8388608, hybrid_huffman_threshold=1000000, output_type=types.RGB,
-              preserve=False, seed=1, split_stages=False, use_chunk_allocator=False, use_fast_idct=False, device=None, 
-              decode_size_policy=types.USER_GIVEN_SIZE_ORIG, max_decoded_width=1000, max_decoded_height=1000):
+def image_raw(*inputs, user_feature_key_map=None, path='', random_shuffle=False, output_type=types.RGB, max_decoded_width=1000, max_decoded_height=1000):
```

Comment:
Where is the decode_size_policy here?

Reply:
No decode size policy used for tfrecord readers